### PR TITLE
Babylon foreign laws rework

### PR DESCRIPTION
**Pull request by HiddeLekanne**
Completely reworks the Foreign laws according to the design document: https://docs.google.com/document/d/1fsiX1q1nO3VRDB7mawTbq2ewAqoMPJNo42IDfz5LibY/edit#

### CODE REVIEW

File: common/ideologies/00_ig_ideologies.txt [hunk 1]
```diff
@@ -486,6 +465,15 @@ ideology_republican = {
...
 		law_anarchy = strongly_disapprove
+	}
+
+	lawgroup_diplomatic_stance = {
+		law_expansionism = neutral
+		law_imperialism = neutral
+		law_armed_neutrality = neutral
+		law_anti_imperialist = neutral
+		law_pacifist = neutral
+		law_true_neutrality = approve
```
Comment (HiddeLekanne) [inline common/ideologies/00_ig_ideologies.txt]: @banana9191 You added it to the wrong law, should be liberal.

File: common/ideologies/00_ig_ideologies.txt [hunk 2]
```diff
@@ -720,6 +689,14 @@ ideology_patriarchal = {
...
 		law_protected_speech = disapprove
 	}
+	lawgroup_diplomatic_stance = {
+		law_expansionism = approve
+		law_imperialism = neutral
+		law_armed_neutrality = approve
+		law_anti_imperialist = neutral
+		law_pacifist = neutral
+		law_true_neutrality = neutral
+	}
```
Comment (HiddeLekanne) [inline common/ideologies/00_ig_ideologies.txt]: @banana9191 You added it to the wrong law it should be paternalistic

File: common/ideologies/00_ig_ideologies.txt [hunk 3]
```diff
@@ -737,13 +714,12 @@ ideology_anti_slavery = {
...
-		law_anti_imperialist = strongly_approve
-		law_neutral = disapprove
-		law_pacifist = strongly_disapprove
-		law_true_neutrality = neutral
+		law_expansionism = neutral
+		law_imperialism = neutral
+		law_armed_neutrality = neutral
+		law_anti_imperialist = neutral
+		law_pacifist = neutral
+		law_true_neutrality = approve
```
Comment (HiddeLekanne) [inline common/ideologies/00_ig_ideologies.txt]: Should have been removed.

Comment (HiddeLekanne) [reply common/ideologies/00_ig_ideologies.txt]: @banana9191

File: common/scripted_effects/babylon_remove_infamy_effects.txt [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+		name = infamy_difference
+		subtract = infamy
+	}
+	change_infamy = local_var:infamy_difference
+}
+
+test_effect = {
+	debug_log = "my stuff"
+	debug_log_scopes = yes
+}
```
Comment (Danarca) [inline common/scripted_effects/babylon_remove_infamy_effects.txt]: Whoops, looks like I left a debug_effect in! Oh well, I don't think it applies anywhere.

File: localization/english/babylon_law_l_english.yml [hunk 5]
```diff
@@ -5,19 +5,24 @@
 
  law_expansionism: "Expansionism"
  law_expansionism_desc: "We are a nation that takes what is ours through strength alone. Borders are simply a show of a nations strength."
+ custom_babylon_law_expansionism_tip: "@green_checkmark! You can conquer or subjugate lands neighbouring your incorperated states regardless of culture"
```
Comment (Danarca) [inline localization/english/babylon_law_l_english.yml]: incorperated -> incorporated

**Review by Danarca — APPROVED** [review]
Single typo from what I saw. 
Otherwise, it looks fine, although I cannot figure out the dp_colonial_war diplomatic play :s

**Review by Danarca — APPROVED** [review]
Can't put my finger on anything else :)
Approved!

**Pull request merged**